### PR TITLE
Improve generation and data flow

### PR DESCRIPTION
**Pull request by rgieseke**
This improves data and generation from Makefiles. CEDS data is now downloaded if necessary and no longer part of the repository.

For a couple of gases we have the combination of CEDS and Biomass Burning emissions.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
 	./venv/bin/pip install --upgrade pip
 	./venv/bin/pip install -Ur requirements.txt
 	touch venv
 
-global-carbon-budget:
-	git clone https://github.com/openclimatedata/global-carbon-budget data/global-carbon-budget
+download: raw-data/global-carbon-budget raw-data/global-carbon-budget-projection.csv raw-data/edgar-co2-emissions raw-data/ceds raw-data/primap-hist raw-data/global-biomass-burning-emissions
+
+raw-data/global-carbon-budget:
+	git -C raw-data/global-carbon-budget pull || git clone https://github.com/openclimatedata/global-carbon-budget raw-data/global-carbon-budget
```
Comment (znicholls) [inline Makefile]: should we silence these so we don't get `can't change to path` error messages?

Comment (rgieseke) [reply Makefile]: Would be nice ...

**Comment by rgieseke** [comment]
All checks have passed, for the paper generation Pyhector shouldn't matter ...?

**Comment by znicholls** [comment]
hmm it's failing nicely on my mac

**Comment by znicholls** [comment]
Fixed I think

File: Makefile (hunk 1 above)
Comment (rgieseke) [reply Makefile]: I'm considering to have everything archived with DOIs -- so these git clones shouldn't be in there in the end.

**Comment by rgieseke** [comment]
> hmm it's failing nicely on my mac

i had some issues on my mac with building the output and getting extra `images/images`  :-/

**Comment by znicholls** [comment]
Shall we merge this then I can deal with extra images on local build later?

**Comment by rgieseke** [comment]
If you could take a look at the unit conversion here 

https://github.com/openclimatedata/global-emissions/pull/12/files#diff-f1cda2be9036b5a29bcf3444466dcb29R67

Otherwise it should be unproblematic ... and ready to be merged for now :-)

**Comment by rgieseke** [comment]
Thanks! I left a few Todos in there so I should definitely do more checks ...

I'll try to fix the merge conflicts ...

**Pull request merged**